src/chipeitlib/src/main/java/es/chipeit/lib/core/ByteMemory.kt

```diff
@@ -2,7 +2,7 @@ package es.chipeit.lib.core
 
 import es.chipeit.lib.interfaces.IMemory
 
-internal class ByteMemory(private val memory: ByteArray): IMemory<Byte> {
+internal open class ByteMemory(protected val memory: ByteArray): IMemory<Byte> {
```

Comment:
No class seems to extend ByteMemory. Revert this change (remove `open` and revert `protected` to `private`).